DQM/SiPixelPhase1TrackClusters/src/SiPixelPhase1TrackClusters.cc

```diff
@@ -211,10 +223,15 @@ void SiPixelPhase1TrackClusters::analyze(const edm::Event& iEvent, const edm::Ev
        histo[SiPixelPhase1TrackClustersOnTrackShape].fill(shape?1:0,id, &iEvent);
       }
 
+      std::cout<<"Cluster charge: "<<charge<<endl;
       for (int i =0; i<cluster.size(); i++){
+	  // get pixel cluster information
           SiPixelCluster::Pixel vecipxl = cluster.pixel(i);
           histo[SiPixelPhase1DigisHitmapOnTrack].fill(id, &iEvent, vecipxl.y, vecipxl.x);
+          histo[SiPixelPhase1PixelChargeOnTrack].fill(vecipxl.adc, id, &iEvent);
       }
+       
+      histo[SiPixelPhase1NPixelsByNDigisOnTrack].fill(cluster.size()/input.product()->size(), id, &iEvent);
```

Comment:
this is not the Histogram I asked for, you should count ALL the digis (i.e the Pixels) that belongs to on-track clusters and at the end of the loop on tracks plot the ratio between the total number of on-track digis in the event and the total number of PixelDigis